### PR TITLE
Feature/WRN-16901: [Moonstone tests] UI Tests Migration

### PR DESCRIPTION
**Pull request by vladut-clapou-lgp**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [X] Documentation was verified or is not changed
* [X] UI test was passed or is not needed
* [ ] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated tests to work asynchronously.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-16901

### Comments
Enact-DCO-1.0-Signed-off-by: Clapou George-Vladut [vladut.clapou@lgepartner.com](mailto:vladut.clapou@lgepartner.com)

### CODE REVIEW

**Comment by vladut-clapou-lgp** [comment]
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [X] Documentation was verified or is not changed
* [X] UI test was passed or is not needed
* [ ] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated tests to work asynchronously.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)


### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)


### Links
[//]: # (Related issues, references)
WRN-16901

### Comments
Enact-DCO-1.0-Signed-off-by: Clapou George-Vladut [vladut.clapou@lgepartner.com](mailto:vladut.clapou@lgepartner.com)

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: tests/ui/specs/Popup/Popup-specs.js [hunk 1]
```diff
@@ -6,185 +6,186 @@ describe('Popup', function () {
...
-			validateTitle(popup, 'Popup with AutoDismiss');
+			await expectOpen(popupCommon);
+			await validateTitle(popup, 'Popup with AutoDismiss');
 		});
 
 		describe('5-way', function () {
 
-			it('should spot default button in popup container', function () {
-				Page.waitTransitionEnd(3000, 'popup open', () => {
+			// it('should spot default button in popup container', async function () {
```
Comment (jeonghee27) [inline tests/ui/specs/Popup/Popup-specs.js]: Popup tc fails is going to be fixed on WRN-17780. currently, there is regression on enact/develop branch. 
Could you migrate these TC using enact/master branch?  
Sorry for inconvenience

Comment (daniel-stoian-lgp) [reply tests/ui/specs/Popup/Popup-specs.js]: they work with enact/master. thanks for the tip

File: tests/ui/specs/Popup/Popup-specs.js [hunk 2]
```diff
@@ -193,1266 +192,1248 @@ describe('Popup', function () {
...
-			it('should not dismiss the popup and should not move spotlight from the popup container', function () {
-				Page.spotlightRight();
-				Page.waitTransitionEnd(3000, 'popup open', () => {
+			it('should spot default button in popup container', async function () {
+				await Page.spotlightRight();
+				// await Page.waitTransitionEnd(3000, 'popup open', async () => {
+				// 	await Page.spotlightSelect();
+				// });
+				await Page.spotlightSelect();
+				await browser.pause(3000);
```
Comment (jeonghee27) [inline tests/ui/specs/Popup/Popup-specs.js]: Can't we use waitTransitionEnd?  Can't it be like line 217?
Because it aways wait 3sec

Comment (daniel-stoian-lgp) [reply tests/ui/specs/Popup/Popup-specs.js]: sorry, i missed this. i pushed a fix

**Pull request merged**